### PR TITLE
refactor(common): move kpj-related files into developer-utils 🐉

### PR DESCRIPTION
**Pull request by mcdurdin**
Part of #9665.

In preparation for moving source file types into developer-utils, move the existing developer-utils files into src/ and src/utils subfolders.

Moves the kpj test fixtures into developer-utils, moves from let to const in code where required by eslint, and reduces the coverage threshold in order to get tests to pass for developer-utils.

As we move some better-tested areas out of common-types, it looks like our overall coverage goes down there, but it _also_ went down in developer-utils! 😭

@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/types/build.sh [hunk 1]
```diff
@@ -100,7 +100,10 @@ function do_build() {
 function do_test() {
   eslint .
   tsc --build test
-  c8 --skip-full --reporter=lcov --reporter=text mocha "${builder_extra_params[@]}"
+  readonly C8_THRESHOLD=75
+  c8 -skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha "${builder_extra_params[@]}"
```
Comment (jahorton) [inline common/web/types/build.sh:104]: `--skip-full` appears to have lost a hyphen here.  Is that intended?

**Review by jahorton — APPROVED** [review]
It looks like this PR mostly moves Developer-only pieces formerly within `common/web/types` to `developer/src/common/web/utils/src`?

LGTM, perhaps outside of one nit.

File: common/web/types/build.sh (hunk 1 above)
Comment (mcdurdin) [reply common/web/types/build.sh:104]:
```suggestion
  c8 --skip-full --reporter=lcov --reporter=text --lines $C8_THRESHOLD --statements $C8_THRESHOLD --branches $C8_THRESHOLD --functions $C8_THRESHOLD mocha "${builder_extra_params[@]}"
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.84-alpha](https://keyman.com/downloads/releases/alpha/18.0.84)

**Pull request merged**